med-web-app-front/src/services/pipeline.service.js

```diff
@@ -12,6 +12,10 @@ class PipelineService {
     savePipeline(jsonConfiguration, description) {
         return axios.post(API_URL + 'save', {jsonConfiguration, description},{ headers: authHeader() });
     }
+
+    deletePipeline(pipelineId) {
+        return axios.delete(API_URL + pipelineId,{ headers: authHeader() });

```

Comment:
Codacy has a fix for the issue: [Replace `········return·axios.delete(API_URL·+·pipelineId,{·headers:·authHeader()·});␍` with `····return·axios.delete(API_URL·+·pipelineId,·{·headers:·authHeader()·});`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534713)
```suggestion
    return axios.delete(API_URL + pipelineId, { headers: authHeader() });
```